src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/metadata/MetadataAjaxController.java

```diff
@@ -23,10 +23,12 @@
 @RequestMapping("/ajax/metadata")
 public class MetadataAjaxController {
 	private final UIMetadataService service;
+	private final MessageSource messageSource;
 
 	@Autowired
-	public MetadataAjaxController(UIMetadataService service) {
+	public MetadataAjaxController(UIMetadataService service, MessageSource messageSource) {
 		this.service = service;
+		this.messageSource = messageSource;
```

Comment:
So if you look at the rest of the controllers in this class, I have been working really hard to keep message source inside the service since it is needed their to internationalize stuff within the responses.

Reply:
Updated in 8e63a642f3f46bf750d3af026924c0835221c272